packages/esm-ward-app/src/ward-workspace/patient-banner/patient-banner.component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+import { useConfig } from '@openmrs/esm-framework';
+import { defaultPatientCardElementConfig, type WardConfigObject } from '../../config-schema';
+import { useParams } from 'react-router-dom';
+import { getPatientCardElementFromDefinition } from '../../ward-patient-card/ward-patient-card-row.resources';
+import type { PatientCardElementType, WardPatientCardProps } from '../../types';
+import styles from './style.scss';
+
+const WardPatientWorkspaceBanner = ({ bed, patient, visit }: WardPatientCardProps) => {
```

Comment:
Will defer to @chibongho  when he gets back on this part...

Reply:
@chibongho I hope you can now take a look at this. Thanks